backend/scripts/start-on-testnet.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Stop and remove containers, networks
+echo "Stopping and removing containers, networks..."
+docker compose -f docker-compose-testnet.yaml --profile full down
+
+# Remove specified volumes to remove all state and start fresh
+echo "Removing specified volumes..."
+docker volume rm backend_redis_data
+docker volume rm backend_chainstorage
```

Comment:
I don't think this line matters right?

Reply:
It's been helpful for local development, definitely don't need the chainstorage part, but we might want a development option to clean up the redis data.